composer.json

```diff
@@ -11,6 +11,15 @@
     "require": {
         "marco-c/wp_serve_file": "dev-master"
     },
+    "require-dev": {
+        "stevegrunwell/wp-enforcer": "^0.4.1"
+    },
+    "post-install-cmd": [
+    	"wp-enforcer"
```

Comment:
@delapuente  Sorry for the late reply. I went ahead and removed those commands, here is what they did: "If you'd like to require WP Enforcer for all developers on your project, you can add the following commands to your composer.json file to have WP Enforcer automatically set up Git hooks on composer install and composer update."